### PR TITLE
chore(HomeWorkList): adds List exercises

### PR DESCRIPTION
**Pull request by KurisuLim**
### Lists:

* Initiate an Arraylist. Using For loop add integers from 1 to 10
* Create an ArrayList [1,2,3,6,7,3,2,1]. Remove all 2 from Arraylist
* Remove all odd numbers;

### CODE REVIEW

File: src/WeekFour/HomeWorkList.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        initArrayList(10); // # 1
+        System.out.println("\n#2\n");
+        removeInt();// # 2
+        System.out.println("\n#3\n");
+        removeOddNum(10);//# 3
+    }
+    public static void initArrayList(int num){
+        int number = num;
+
+        List<Integer> array_list = new ArrayList();
```
Comment (vladimirKorzhukov) [inline src/WeekFour/HomeWorkList.java]: Please use "camalCase" for variables.

File: src/WeekFour/HomeWorkList.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        }
+        System.out.println("\nPrinting the array list\n");
+        for(int list : array_list){
+            System.out.println("output: " + list);
+        }
+
+    }
+
+    public static void removeInt(){
+        List<Integer> array_num = new ArrayList<>();
```
Comment (vladimirKorzhukov) [inline src/WeekFour/HomeWorkList.java]: Please use "camalCase" for variables.

File: src/WeekFour/HomeWorkList.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        System.out.println("Current array list: " + array_num);
+        array_num.removeAll(Arrays.asList(2));
+        System.out.println("Removing the number: " + 2);
+        System.out.println("Current array list: " + array_num);
+
+    }
+    public static void removeOddNum(int num){
+        int number = num;
+
+        List<Integer> array_num = new ArrayList<>();
```
Comment (vladimirKorzhukov) [inline src/WeekFour/HomeWorkList.java]: Please use "camalCase" for variables.

File: src/WeekFour/HomeWorkList.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+        for(int i = 1; i <= number; i++){
```
Comment (vladimirKorzhukov) [inline src/WeekFour/HomeWorkList.java:54]: This is a good solution but you have to initialize and add numbers to the list and then remove all odd numbers from the list using "remove" method

**Comment by KurisuLim** [comment]
@vladimirKorzhukov Please recheck.

**Pull request merged**